plugins/outputs/event_hubs/event_hubs.go

```diff
@@ -20,15 +21,12 @@ import (
 var sampleConfig string
 
 type EventHubs struct {
-	ConnectionString string          `toml:"connection_string"`
-	PartitionKey     string          `toml:"partition_key"`
-	MaxMessageSize   config.Size     `toml:"max_message_size"`
-	Timeout          config.Duration `toml:"timeout"`
-	Log              telegraf.Logger `toml:"-"`
+	eh.Config
 
 	client     *azeventhubs.ProducerClient
 	options    azeventhubs.EventDataBatchOptions
 	serializer telegraf.Serializer
+	log        telegraf.Logger `toml:"-"`
```

Comment:
No, this field needs to be exported as otherwise it will not be initialized by Telegraf.